language-design.md

```diff
@@ -17,7 +17,34 @@ For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
 
-
+> We should not make the Java programming language a cathedral, but a plain
+> bazaar might be too loose. What we need is more like a shopping mall, where there are
+> not quite as many choices but most of the goods are well designed and sellers stand up
+> and back what they sell. [Steele, 1998]
+
+We both agree with this quote for the most part, but we are a bit concerned that sometimes maintainers would disappear. If languages lean slightly cathedral from a shopping mall model, then they can have patterns for dealing with this problem. Real-life shopping malls kick vendors out if they stop paying rent, but this doesn't work when "Public APIs, like diamonds, are forever" [Bloch, 2006].
+
+> A small programming language might take but a short time to learn. 
+> A large programming language may take a long, long time to learn, 
+> but then it is less hard to use, for we then have a lot of words at hand [Steele, 1998]
+
+We both agree with this in general, though it is possible to construct the syntax in a way
+that would make the language either easier or more difficult to learn regardless of size.
+In Python, the language is small, but there is a large library, so there are ways to have
+"a lot of words at hand" without making a larger language. Perl has complex syntax and 
```

Comment:
How do we define the size of a language, though? Is it the number of words included in the base languages, no libraries added, or the base language plus common libraries? I would imagine that by the definition you're using, C++ wouldn't be a particularly big language either, right? Are there any languages that has a **lot** of words included in the base language itself?